README.md

```diff
@@ -28,15 +28,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. What is the DOM?
+1. What is the DOM? An abstaction that represents the elements that make up the website, including the HTML, styling and javascript. By looking at them in the DOM, we can analyze and manipulate the DOM instead of the individual pieces.
 
 2. What is an event?
 
+An event is a specific action, usually, but not always initiated by the user that will trigger a component or module that performs a specific action or function.  An example of an event would be a click on a button that saves or minimizes the page.
+
 3. What is an event listener?
 
+An event listener is a code module or component that is listening for the event and is executed when it detects the event. 
+
```

Comment:
Almost an event listener is not a component it's actually a function that you can put on an element that takes in an event as a parameter